auth/src/main/res/values-b+es+419/strings.xml

```diff
@@ -50,17 +50,11 @@
     <string name="fui_resend_code_in">Se reenviará el código en 0:%02d</string>
     <string name="fui_verify_your_phone_title">Verifica tu número de teléfono</string>
     <string name="fui_verifying">Verificando…</string>
-    <string name="fui_retrieving_sms">Recuperando SMS…</string>
     <string name="fui_incorrect_code_dialog_body">Código incorrecto. Vuelve a intentarlo.</string>
-    <string name="fui_incorrect_code_dialog_positive_button_text">Aceptar</string>
     <string name="fui_error_too_many_attempts">Este número de teléfono se usó demasiadas veces</string>
     <string name="fui_error_quota_exceeded">Ocurrió un problema durante la verificación de tu número de teléfono</string>
     <string name="fui_error_session_expired">Este código ya no es válido</string>
     <string name="fui_sign_in_with_phone_number">Acceder con el número de teléfono</string>
-    <string name="fui_done">Listo</string>
```

Comment:
This is definitely a highly annoying request, but could we split out the string removals into another PR?  That way this one doesn't have 177 files changed and thousands of lines.  I don't want an important behavioral change to get lost among all of this.

Reply:
Not at all, I thought you might ask so I removed all the strings in one easily rebasable commit. 👍